src/main/java/mate/academy/rickandmorty/service/CharacterServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.rickandmorty.service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.CharacterRequestDto;
+import mate.academy.rickandmorty.dto.CharacterResponseDto;
+import mate.academy.rickandmorty.mapper.CharacterMapper;
+import mate.academy.rickandmorty.repository.CharacterRepository;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class CharacterServiceImpl implements CharacterService {
+    private final CharacterRepository repository;
+    private final CharacterMapper mapper;
```

Comment:
```suggestion
    private final CharacterRepository characterRepository;
    private final CharacterMapper characterMapper;
```
make more descriptive the var names